Samples.xcodeproj/project.pbxproj

```diff
@@ -756,14 +771,15 @@
 		00CCB8A6285D059300BBAB70 /* Portal Data */ = {
 			isa = PBXGroup;
 			children = (
-				D701D7242A37C7E4006FF0C8 /* 07d62a792ab6496d9b772a24efea45d0 */,
```

Comment:
These changes are expected. For some reason, the directories weren't sorted in the portal data folder.